graph/edge/impl/TypeEdgeImpl.java

```diff
@@ -344,6 +379,69 @@ public void commit() {
      */
     public static class Persisted extends TypeEdgeImpl implements TypeEdge {
 
+        public static class EncodedValue {
+            private static final byte KEY_OVERRIDEN = 1;
+            private static final byte KEY_UNIQUE = 2;
+            // We used to store the overridden type directly
+            //  Encoding.prefix.VERTEX_ATTRIBUTE_TYPE.key = 111;
+            //  Encoding.prefix.VERTEX_ROLE_TYPE.key = 113;
```

Comment:
Need a better way to mark that these are expected values.
I'm afraid of using `private static final byte KEY_LEGACY_ROLE_TYPE = Encoding.prefix.VERTEX_ROLE_TYPE.key()` and introducing static initialisation dependencies.